src/sentry/api/serializers/models/filechange.py

```diff
@@ -13,22 +13,28 @@ def get_attrs(self, item_list, user):
         commits = Commit.objects.filter(id__in=[f.commit_id for f in item_list]).select_related('author')
         author_objs = get_users_for_commits(commits)
         commits_by_id = {commit.id: commit for commit in commits}
+
+        repositories = Repository.objects.filter(id__in=[commit.repository_id for commit in commits])
+        repo_names_by_id = {repository.id: repository.name for repository in repositories}
+
         result = {}
         for item in item_list:
             commit = commits_by_id[item.commit_id]
             result[item] = {
                 'user': author_objs.get(commit.author_id, {}),
-                'message': commit.message
+                'message': commit.message,
+                'repository_name': repo_names_by_id[commit.repository_id]
```

Comment:
I'd probably guard against a missing key here.

```python
repo_names_by_id.get(commit.repository_id)
```

I'm not entirely sure if this case will be hit, but there's no reason it couldn't happen. There isn't a foreign key link here, so it's possible they can become decoupled.